Units/parser-javascript.r/js-es6-class.d/input.js

```diff
@@ -1,18 +1,18 @@
 
 class Class1
 {
-  method1(arg1,arg2)
-  {
-  }
+	method1(arg1,arg2)
+	{
+	}
   
-  method2(arg1,arg2)
-  {
-  }
+	method2(arg1,arg2)
+	{
+	}
 }
 
 class Class2
 {
-  method3(arg = 10)
-  {
-  }
+	static method3(arg = 10)
+	{
+	}
```

Comment:
About input files of test case, I don't care about whitespaces. 
However, I wonder why these indentation changes are needed.

I take following workflow to avoid injecting this kind of changes:

1. Do git commit
2. See the output of git show and find whether I introduced unwanted change(unnecessary change) in the last commit or not.
3. If I found one, remove it with my editor.
4. Do git add, git commit --amend. (or mark `fixup` in git rebase -i); then Goto 2.
5. Make a pull request